apps/ewallet_db/lib/ewallet_db/role.ex

```diff
@@ -35,15 +35,15 @@ defmodule EWalletDB.Role do
   Creates a new role with the passed attributes.
   """
   def insert(attrs) do
-    last_role = Role |> Query.last(:inserted_at) |> Repo.one()
+    highest_priority = Repo.one(from(r in Role, select: max(r.priority)))
```

Comment:
Was thinking of `lowest_priority` but `lowest_priority` could also mean lowest value, which has the high privileges 🤦‍♂️ Couldn't think of a better name at the moment.